source/_patterns/04-pages/research-article/research-article.yaml

```diff
@@ -40,10 +40,16 @@ body:
           angiosperm origins.
   - template: atoms-paragraph
     arguments:
-      text: |
-        From oranges to apples, flowering plants produce most of the fruits and vegetables that we can see on display in
-        a supermarket. While we may take little notice of the poppy fields and plum blossoms around us, how flowers came
-        to be has been an intensely debated mystery.
+      text:
+        - |
```

Comment:
This syntax allows strings to be split over multiple lines, I'd only use it when needed.